financialaid/views_test.py

```diff
@@ -0,0 +1,75 @@
+"""
+Tests for financialaid view
+"""
+from django.core.urlresolvers import reverse
+from django.db.models.signals import post_save
+
+from factory.django import mute_signals
+from rest_framework.status import HTTP_201_CREATED, HTTP_400_BAD_REQUEST
+from rest_framework.test import APIClient
+
+from financialaid.backend_tests import FinancialAidBackendTests
+from financialaid.models import FinancialAid
+from profiles.factories import ProfileFactory
+
+
+class FinancialAidViewTests(FinancialAidBackendTests, APIClient):
```

Comment:
you should not import here another test class `FinancialAidBackendTests`

Instead create a common base class and import that one here